miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppPermission.kt

```diff
@@ -22,9 +22,11 @@ enum class MiniAppCustomPermissionType(val type: String) {
     LOCATION("rakuten.miniapp.device.LOCATION"),
     UNKNOWN("UNKNOWN");
 
-    internal companion object {
+    companion object {
 
-        internal fun getValue(type: String) = values().find { it.type == type } ?: UNKNOWN
+        /** get the value of custom permission per type. **/
+        fun getValue(type: String): MiniAppCustomPermissionType =
```

Comment:
Why do we need to make it public here? I think only miniapp sdk use it so that's why we made it internal.